pkg/virt-controller/services/template_test.go

```diff
@@ -4070,7 +4070,7 @@ var _ = Describe("Template", func() {
 			Expect(memLimit).To(Equal(expMemLimitQ.Value()))
 			expCpuReqQ := resource.MustParse("100m")
 			Expect(cpuReq).To(Equal(expCpuReqQ.Value()))
-			expMemReqQ := resource.MustParse("80M")
+			expMemReqQ := resource.MustParse("2M")
```

Comment:
I'd expect `expCpuReqQ := resource.MustParse("100m")` to be `expCpuReqQ := resource.MustParse("10m")`

Reply:
Sorry, reading my own comment I realize this is too criptic.
What I mean is that here the expected memory request was reduced back to `2M`.
I would expect the same to happen for the cpu request too, IOW to be rolled back to `10m`, but it is not.
Can you explain? Thanks

Reply:
No you are right, have to check why just the one, and not the other.

Reply:
So turns out the unscaled value of 10m and 100m is both '1' and thus matches, instead I need to use MilliValue, and that fixes the issue.